### PR TITLE
Akibr 456 create project info data model

### LINKED ISSUE #456: Create ProjectInfo data model
The task is blocked by:

- [x] #454 
- [x] #455 
- [x] #457 

### Overview
`ProjectInfo` is a new model for project representation in the database. We need to create a separate schema of the project to not carry the burden of maintenance legacy Project records in parallel with new ones.

Create a Mongoose schema `ProjectInfo` containing the following properties:

```
const TYPES = ['Software', 'Org-wide', 'Community of practice'];
const STATUSES = ['Active', 'Inactive'];

name: string		// project name or team’s title
type: string, enum TYPES
description: string, length limit of 1000 symbols
locationZone: string, validated by LocationModel from #454 (see the example link in Resources)
timeZone: string, validated by TimeZoneModel from #455 
partner: string		// external organization partnering to sponsor project
partnerUrl: string	// URL of partner organization
activeStatus: string, enum STATUSES
defaultConferenceUrl: string  // project’s Zoom/Google Meet/etc. link
technologies: array of strings		// project’s technologies in use for user skills matching
languages: array of strings		// project’s programming languages in use for user skills matching
mainImage: Buffer		// project’s main or cover image
mainImageText	// alternative text for main/cover image
backgroundImage: Buffer	// project page hero/top-of-page background image for website
backgroundImageText	// alternative text for background image
urls: {   // all fields are strings
    googleDrive	// project’s top-level Google Drive URL
    slack		        // project’s main slack channel URL
    repository		// project’s GitHub URL
    readme		// project’s GitHub Readme URL
    wiki	        	// project’s GitHub Wiki URL
    hflaPage		// project’s HfLA website dedicated page URL
    website		// project’s public independent website URL
    otherUrlLabel    	// additional URL label
    otherUrlValue   	// additional URL value
}
seekingRoles: array of strings, validated by JobRoles from #457 (see the example link in Resources)		// roles project is currently recruiting for
recruitingDetails: string, max 1000 symbols     // additional recruiting details

created: date
lastModified: date
completed: date
```

### Action Items

- [ ] Create a schema
- [ ] Create the schema test

### Resources
[Models directory](https://github.com/hackforla/VRMS/tree/development/backend/models)
[Custom validator example](https://stackoverflow.com/questions/43159336/what-to-do-with-enum-values-in-mongoose)

### PR DESCRIPTION
**Pull request by akibrhast**
Closes #456 #493
- fix typo in models/dictionaries/index.js
- remove console.log from models/projectInfo.test.js
- rewrite validation in models/projectInfo.model.js

### CODE REVIEW

File: backend/models/projectInfo.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    expect(savedData.urls.repository).toBe(submittedData.urls.repository);
+
+
+    done();
+  });
+});
+
+describe('CREATE/READ', () => {
+  test('Create ProjectInfo with Mongoose model', async (done) => {
+    // const submittedData = {
```
Comment (alex-anakin) [inline backend/models/projectInfo.test.js]: Commenting out tests is not a fix. Tests failed because of some code issues not because they are bad.

**Review by alex-anakin — CHANGES_REQUESTED** [review]
Please apply a fix that will make tests pass.

File: backend/models/projectInfo.test.js (hunk 1 above)
Comment (akibrhast) [reply backend/models/projectInfo.test.js]: @alex-anakin I actually did not comment that out. Those were originally commented out in https://github.com/hackforla/VRMS/pull/493/files

Comment (akibrhast) [reply backend/models/projectInfo.test.js]: I have uncommented the pertinent section as requested

**Review by alex-anakin — APPROVED** [review]
Looks good!

**Pull request merged**